CMakeCommon.cmake

```diff
@@ -101,6 +101,9 @@ endfunction ()
 
 function (GenerateAddOnProject target acVersion devKitDir addOnName addOnSourcesFolder addOnResourcesFolder addOnLanguage)
 
+    verify_api_devkit_folder (${devKitDir})
```

Comment:
For new code, please avoid unintentional list expansion of variables. To avoid list expansion, make sure the variable substitution is quoted: `"${devKitDir}"`

Please apply to the other instances within the additions as well.